### DISCUSSION TITLE
[QUESTION]Audio sampling rate mismatch

### DISCUSSION THREAD

**Discussion by yy-arya** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1693] [Q&A]
Hi, 
Using webrtc, when the sampling rate is set to 8000, the audio is normal. When the master send audio of the sampling rate 16000, the browser end still processes it with 8000, which causes abnormal audio playback. How to unify the audio sampling rate on both ends? How should I set it?

Do you have any suggestions for me? thank you!

**Comment by disa6302** [comment]
@yy-arya , 

What are you running as the viewer on the browser?

**Reply by yy-arya** [reply]
Sorry, I don't quite understand what your question means. could you be more specific?
I open the viewer on Google Chrome and receive audio and video.
